### PR TITLE
Add contributors

### LINKED ISSUE #38: Acknowledge contributors
All contributors should be acknowledged

- [ ] In package.json
- [ ] In the README (something like https://github.com/all-contributors/all-contributors)

### PR DESCRIPTION
**Pull request by Joezo**
Adds contributors to package json and readme.

closes #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,3 +320,9 @@ Coming soon!
 ### Migrating from Apollo
 
 Coming soon!
+
+## Contributors
+
+<!-- prettier-ignore -->
+| <img src="https://avatars.githubusercontent.com/u/1939483?v=3" width="100px;" alt="Brian Mullan"/> <br /> [Brian Mullan](https://github.com/bmullan91) | <img src="https://avatars.githubusercontent.com/u/1485654?v=3" width="100px;" alt="Jack Clark"/> <br /> [Jack Clark](https://github.com/jackdclark) | <img src="https://avatars.githubusercontent.com/u/2870255?v=3" width="100px;" alt="Joe Warren"/> <br /> [Joe Warren](https://github.com/joezo) | <img src="https://avatars.githubusercontent.com/u/20181?v=3" width="100px;" alt="Simone Busoli"/> <br /> [Simone Busoli](https://github.com/simoneb) |   |
```
Comment (jackdclark) [inline README.md]:
```suggestion
| <img src="https://avatars.githubusercontent.com/u/1939483?v=3" width="100px;" alt="Brian Mullan"/> <br /> [Brian Mullan](https://github.com/bmullan91) | <img src="https://avatars.githubusercontent.com/u/1485654?v=3" width="100px;" alt="Jack Clark"/> <br /> [Jack Clark](https://github.com/jackdclark) | <img src="https://avatars.githubusercontent.com/u/2870255?v=3" width="100px;" alt="Joe Warren"/> <br /> [Joe Warren](https://github.com/joezo) | <img src="https://avatars.githubusercontent.com/u/20181?v=3" width="100px;" alt="Simone Busoli"/> <br /> [Simone Busoli](https://github.com/simoneb) |
```

File: README.md [hunk 2]
```diff
@@ -320,3 +320,9 @@ Coming soon!
...
+|---|---|---|---|---|
```
Comment (jackdclark) [inline README.md]:
```suggestion
|---|---|---|---|
```

**Comment by Joezo** [comment]
It is not automated. I'm happy to add that bot though?

**Comment by bmullan91** [comment]
@all-contributors please add @bmullan91

**Comment by Joezo** [comment]
@all-contributors please add @bmullan91 for doc

**Comment by Joezo** [comment]
heh, that's quite cool. Let's use that!

**Pull request closed without merging**